src/View/Helper/FlashMessenger.php

```diff
@@ -369,4 +407,19 @@ protected function getEscapeHtmlHelper()
 
         return $this->escapeHtmlHelper;
     }
+
+    private function getNamespace(string $namespace): ?FlashMessengerNamespace
+    {
+        return $this->namespaces[$namespace] ?? null;
+    }
+
+    private function getClasses(string $namespace): string
+    {
+        $objNamespace = $this->getNamespace($namespace);
```

Comment:
To avoid `$objNamespace` (which is hungarian notation), let's instead use `string $namespaceName` as input parameter, and then just `$namespace` for the object form?